torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -156,6 +156,78 @@ def fuse(self, graph, subset):
             graph.erase_node(original_mm)
 
 
+class BatchLayernormFusion(BatchFusion):
+    """
+    Batch layer norm fusion in pre grad pass
+    """
+
+    def match(self, node):
+        if CallFunctionVarArgs(torch.nn.functional.layer_norm).match(node):
+            input_node = get_arg_value(node, 0, "input")
+            group_key = (
+                "batch_layernorm",
+                str(input_node.meta["example_value"].shape)
+                if "example_value" in input_node.meta
+                else "",
```

Comment:
What happens with missing example values?  Should this prevent fusion?

Reply:
Make sense. Should prevent such fusions.